todo.go

```diff
@@ -128,3 +128,46 @@ func (t *Todos) Print() {
 	table.SetStyle(simpletable.StyleUnicode)
 	table.Println()
 }
+
+// Add a new function to filter out pending tasks
+func (t *Todos) PrintCompleted() {
+	table := simpletable.New()
+
+	table.Header = &simpletable.Header{
+		Cells: []*simpletable.Cell{
+			{Align: simpletable.AlignCenter, Text: "#"},
+			{Align: simpletable.AlignCenter, Text: "Task"},
+			{Align: simpletable.AlignCenter, Text: "Done?"},
+			{Align: simpletable.AlignCenter, Text: "CreatedAt"},
+		},
+	}
+
+	var cells [][]*simpletable.Cell
+
+	for idx, item := range *t {
+		if item.Done {
+			idx++
+			task := blue(item.Task)
+			done := blue("no")
+			if item.Done {
+				task = green(fmt.Sprintf("\u2705 %s", item.Task))
+				done = green("yes")
+			}
+			cells = append(cells, *&[]*simpletable.Cell{
+				{Text: fmt.Sprintf("%d", idx)},
+				{Text: task},
+				{Text: done},
+				{Text: item.CreatedAt.Format(time.RFC822)},
+			})
+		}
+	}
+
+	table.Body = &simpletable.Body{Cells: cells}
+
+	table.Footer = &simpletable.Footer{Cells: []*simpletable.Cell{
```

Comment:
No need of this footer here. The footer is supposed to be in main list where it shows the no of pending tasks. Here maybe we can show number of completed tasks in green.